### ISSUE TITLE
VCF Support

### ISSUE THREAD

**Issue by ahwagner** [ga4gh/vrs-python#65]
Conversation during our late October VR call confirmed that we are more interested in extracting VRS Alleles from VCF for use in search than we are recreating VCFs from VRS objects + annotations.

Our VCF parsing tooling therefore should focus on:

- collecting and normalizing Alleles from each record
- reporting VRS Alleles observed in each sample across entire VCF

**Comment by larrybabb** [comment]
Before we can collect and normalize Alleles for each record, we should clarify what is a a `record`. If we are going for the typical VCF with SNVs and small InDels only (not SVs or CNVs) then is each record a Sequence Location, a Set of Alleles or one or more Genotypes (in the case of joint called VCFs).  If we are going to focus on the Alleles then do we represent both the Ref and the Alt allele when it is a heterozygous call? If it's a compound het then do we NOT represent the Ref Allele?

Finally how do we handle VRS normalization if there are multiple Alleles on a `record`?

For example 

```
#CHROM POS      ID         REF   ALT     QUAL   FILTER  INFO            FORMAT      NA00001
20     1110696  rs6040355  A     G,T     67     PASS    NS=2;DP=10;AA=T GT:GQ:DP    1|2:21:6:23,27
20     1234567  microsat1  GTC   G,GTCT  50     PASS    NS=3;DP=9;AA=G  GT:GQ:DP    0/1:35:4
```
In the first line above the A>G and A>T changes could end up normalizing to different sequence locations. Should these be two separate Alleles? Should these be combined as co-occuring variations (aka genotype)? Should Genotypes always share the same sequence location?  Do we need to consider how normalizing the REF allele might impact normalization outcomes that cause the sequence locations from two alleles in the same genotype to be mismatched? Is it possible that our normalization will expand the sequence locations on a given allele that two alleles in a Haplotype might end up overlapping?

Lots of questions to unpack here.

**Comment by larrybabb** [comment]
tagging @wesleygoar so that he can potentially refine the focus of this issue and/or close it based on the recent VCF/VRS work he's done and because of the emergence of the 1.3 Genotype model being proposed.

This issue is pretty vague in terms of having a specific problem to solve, so redefining it or closing it and linking it to something more concrete would be much more useful

**Comment by wesleygoar** [comment]
@larrybabb & @ahwagner based on Alex's original issue we have accomplished this task and have already folded the VCF annotator into the main branch. On the other hand, with the updates we have made to genotype we could expand upon the annotation script if desired.

**Comment by larrybabb** [comment]
IMO we should close this and open a separate ticket for the "expanded genotype requirements". I'm a big fan of keeping the issues constrained and focused. And closing them out asap.

**Comment by ahwagner** [comment]
I agree with @larrybabb.

@wesleygoar will you close this and create the new ticket as proposed?
